_posts/2019-08-06-serverless-static-sites.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "Bringing Serverless to a webpage near you with Hugo & Kubernetes"
+description: With the simplicity of OpenFaaS's templating system we can easily create new templates in no time. Today we will be showing how to continuosly deploy static sites using a new template for hugo and the FunctionIngress CRD for custom domain names.
+date: 2019-08-06
+image: /images/2019-serverless-static-sites/background.jpg
+categories:
+  - community
+  - end-user
+  - ingress
+  - kubernetes
+  - hugo
+author_staff_member: matias
+dark_background: true
+---
+
+Learn how you can continuosly deploy static sites with Hugo, OpenFaaS Cloud (or GitLab) and the FunctionIngress CRD.
```

Comment:
Please paste your blog into a Google Doc to check for spelling mistakes i.e `continuosly` -> `continuously`. I also make typos and spelling mistakes, but this should always be the first port of call before submitting.

Reply:
Yes, sorry about that